src/overlays/actors/ovl_En_Bba_01/z_en_bba_01.c

```diff
@@ -33,84 +42,296 @@ const ActorInit En_Bba_01_InitVars = {
-    /* Fire arrow     */ DMG_ENTRY(0, 0x0),
-    /* Ice arrow      */ DMG_ENTRY(0, 0x0),
-    /* Light arrow    */ DMG_ENTRY(0, 0x0),
-    /* Goron spikes   */ DMG_ENTRY(0, 0x0),
-    /* Deku spin      */ DMG_ENTRY(0, 0x0),
-    /* Deku bubble    */ DMG_ENTRY(0, 0x0),
-    /* Deku launch    */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x12   */ DMG_ENTRY(0, 0x0),
-    /* Zora barrier   */ DMG_ENTRY(0, 0x0),
-    /* Normal shield  */ DMG_ENTRY(0, 0x0),
-    /* Light ray      */ DMG_ENTRY(0, 0x0),
-    /* Thrown object  */ DMG_ENTRY(0, 0x0),
-    /* Zora punch     */ DMG_ENTRY(0, 0x0),
-    /* Spin attack    */ DMG_ENTRY(0, 0x0),
-    /* Sword beam     */ DMG_ENTRY(0, 0x0),
-    /* Normal Roll    */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, 0x0),
-    /* Unblockable    */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, 0x0),
-    /* Powder Keg     */ DMG_ENTRY(0, 0x0),
+static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
+
+static DamageTable sDamageTable = {
+    /* Deku Nut       */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku Stick     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Horse trample  */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Explosives     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Zora boomerang */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Normal arrow   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x06   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Hookshot       */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Goron punch    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Sword          */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Goron pound    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Fire arrow     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Ice arrow      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Light arrow    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Goron spikes   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku spin      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku bubble    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku launch    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x12   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Zora barrier   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Normal shield  */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Light ray      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Thrown object  */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Zora punch     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Spin attack    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Sword beam     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Normal Roll    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Unblockable    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Powder Keg     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
```

Comment:
Opinion: Kinda feel like we should have one generic `DMG_EFF_NONE` for all actors to use otherwise by the time we document every damage table we'll have > 400 copies of zero

Reply:
I wouldn't even bother making an enum/defines for this sort of trivial damagetable.